mllib/src/test/scala/org/apache/spark/mllib/tree/GradientBoostedTreesSuite.scala

```diff
@@ -51,8 +53,9 @@ class GradientBoostedTreesSuite extends SparkFunSuite with MLlibTestSparkContext
             gbt, GradientBoostedTreesSuite.data.toImmutableArraySeq, 0.06)
         } catch {
           case e: java.lang.AssertionError =>
-            logError(s"FAILED for numIterations=$numIterations, learningRate=$learningRate," +
-              s" subsamplingRate=$subsamplingRate")
+            logError(log"FAILED for numIterations=${MDC(NUM_ITERATIONS, numIterations)}, " +
```

Comment:
Let's create a method to reduce duplicated code in this file.

Reply:
Done